docs/DeveloperGuide.adoc

```diff
@@ -767,16 +841,17 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-{More to be added}
-
 [appendix]
 == Non Functional Requirements
 
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-{More to be added}
+.  Should be a portable application (ie. there is no need for an installer to use it).
+.  JAR file should not exceed 50 MB.
+.  Test coverage should reach 85% at any time.
+.  Should always favour security over efficiency in development.
+.  Any user who understands English should be able to use the application with the help of the <<UserGuide#, User Guide>>.
```

Comment:
As mentioned, consider adding an NFR pertaining to the availability of an internet connection.

Reply:
Sure, noted with thanks.